app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -77,10 +73,58 @@ public static SessionStore get() {
         GeckoSession mSession;
     }
 
+    private static class GeckoError {
+
+        public static final String ERROR_URL = "resource://android/assets/html/error/netError.xhtml";
+
+        public static final String NET_ERROR_TYPE = "about:neterror";
```

Comment:
infinite-redirect loops (a page that redirects to itself) do not display the expected error page/text:

![image](https://user-images.githubusercontent.com/203725/43291577-d06060bc-90e6-11e8-954a-8372a498d885.png)
<img width="1530" alt="infinite_redirect_screen shot 2018-07-26 at 2 48 47 pm" src="https://user-images.githubusercontent.com/203725/43291598-eb82720e-90e6-11e8-93d1-502441743990.png">